secboot/encrypt_sb.go

```diff
@@ -45,14 +41,6 @@ func FormatEncryptedDevice(key EncryptionKey, label, node string) error {
 		// enough room
 		MetadataKiBSize:     metadataKiBSize,
 		KeyslotsAreaKiBSize: keyslotsAreaKiBSize,
-
-		// Use fixed parameters for the KDF to avoid the
-		// benchmark. This is okay because we have a high
-		// entropy key and the KDF does not gain us much.
-		KDFOptions: &sb.KDFOptions{
```

Comment:
Has Chris added this already to the new API? We need to keep this for one of our customers unfortunately.

Reply:
@mvo5 @chrisccoulson 

Please see the detailed commit description describing what was done in secboot.

https://github.com/snapcore/snapd/pull/10715/commits/dde591333ade7fd021fc2ad17519e1975f4476af

Please let me know if what was done in secboot is sufficient, or not.

Reply:
Seems to be the same to me

Reply:
Thanks, I think we need https://github.com/snapcore/secboot/pull/174 in seccboot and then use that. If we drop the KDFOption we will regress quite a bit in install time on some devices.

Reply:
I update a PR for you in https://github.com/xnox/snapd/pull/1 - I can also directly merge in here but that felt a little bit impolite.

Reply:
Ok secboot one landed now! great. Let me review what you did here, and we should be to land one thing or another.